app/src/test/java/ar/scacchipa/twittercloneapp/repository/AuthorizationRepositoryTest.kt

```diff
@@ -0,0 +1,97 @@
+package ar.scacchipa.twittercloneapp.repository
+
+import ar.scacchipa.twittercloneapp.datasource.IAuthDataSource
+import ar.scacchipa.twittercloneapp.datasource.UserAccessToken
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import okhttp3.MediaType
+import okhttp3.ResponseBody
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import retrofit2.Response
+
+@ExperimentalCoroutinesApi
+class AuthorizationRepositoryTest {
+
+    private var authorizationRepository: IAuthorizationRepository? = null
```

Comment:
should not be nullable in my opinion. Also please name the variable under testing with the name "subject"

Reply:
Both were fixed.